cypress/integration/articleFlows/commentOnArticle.spec.js

```diff
@@ -253,7 +258,7 @@ describe('Comment on articles', () => {
       cy.findByRole('link', { name: /Reply/ });
 
       cy.findByTestId('comments-container').within(() => {
-        cy.findByLabelText('Toggle dropdown menu').click();
+        cy.findByRole('button', { name: 'Toggle dropdown menu' }).click();
```

Comment:
For the other flaky test, explicilty getting it by the button role seems to do the trick.